packages/dsh-tauri-panel-placeholder/src/client/panel.tsx

```
@@ -0,0 +1,61 @@
+} from './constants'
+import { Content } from './content'
+import { IconPlaceholder } from './icons'
+
+/**
+ * panel.tsx — 面板区（sidebar.panel.action 槽）样板条目：
+ * 「定时任务」。按 dsh-tauri-panel 面板协议接入（见 ../dsh-tauri-panel/PROTOCOL.md）：
+ * 条目用宿主导出的 <ActionItem> 组装（样式/折叠/active 态宿主承担），点击调
+ * 宿主导出的 renderPanelContent 切换会话区替换——本插件零机制代码。
+ */
+
+function PlaceholderPanel(props: PlaceholderPanelProps): ReactElement {
```

ESLint

[error] Fast refresh only works when a file only exports components. Move your component(s) to a separate file. If all exports are HOCs, add them to the `extraHOCs` option.

(react-refresh/only-export-components)